docker-compose.prod.override.yml

```
@@ -0,0 +1,92 @@
+      - SERVICES=sqs
+      - DEBUG=1
+    volumes:
+      - localstack_data:/var/lib/localstack
+      - ./infrastructure/docker/php/init-aws.sh:/etc/localstack/init/ready.d/init-aws.sh
+
+volumes:
+  php_socket:
+  caddy_data:
+  caddy_config:
+  db_data:
+  localstack_data:
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)